### ISSUE TITLE
Create the ceremony animation

### ISSUE THREAD

**Issue by konifar** [DroidKaigi/conference-app-2017#14]
## Overview (Required)
I want to show cool ceremony animation like fireworks.
When DroidKaigi will be started, the animation will be shown.

**Comment by NikoYuwono** [comment]
This one sounds interesting, will try to make a prototype this week and post it here!

**Comment by konifar** [comment]
Great! Thanks 👏

**Comment by konifar** [comment]
By the way, I have an image like Confetti animation on Facebook app.

**Comment by NikoYuwono** [comment]
I see, in which page do you plan to put this animation?

**Comment by amay077** [comment]
Would you like the lottie-android by AirBnB?

* [airbnb/lottie-android: Render After Effects animations natively on Android and iOS](https://github.com/airbnb/lottie-android)

**Comment by NikoYuwono** [comment]
Yes saw that too last week but too bad I can't use adobe after effect :( Planning to use particle based animation!

**Comment by amay077** [comment]
Uh-huh, I can't use after effect too. 😅

**Comment by NikoYuwono** [comment]
Sorry for the late update! Because @konifar -san said He has an image like Confetti animation, I tried to create a sample before creating PR to make sure we are in the same page :smiley:   
Let me know what do you think! And I would be really happy if someone can help me with the assets!

![ezgif com-bbe671d2c0](https://cloud.githubusercontent.com/assets/1751188/22978163/f58c624c-f3d4-11e6-9d78-970c48597831.gif)

**Comment by konifar** [comment]
@NikoYuwono :cool: Thanks for tackling!
> I would be really happy if someone can help me with the assets!

I'm not sure I can help, but I want to! 👍 
Can you share this sample project code?

**Comment by NikoYuwono** [comment]
OK, will upload it tonight since it's in my home PC!

**Comment by NikoYuwono** [comment]
@konifar Uploaded! https://github.com/NikoYuwono/ConfettiDemo  
Currently there are only [three colors](https://github.com/NikoYuwono/ConfettiDemo/tree/master/app/src/main/res/drawable), feel free to replace or add more colors :)

**Comment by konifar** [comment]
@NikoYuwono Thanks! I'll check it!
